### DISCUSSION TITLE
Problemas ao entrar no site da DIO.

### DISCUSSION THREAD

**Discussion by Marian97a** [digitalinnovationone/dio-lab-open-source#31213] [Perguntas e Respostas] [answered]
Olá, Bom dia 
Estou com problemas para entrar no site da DIO. ao tentar fazer o login so fica inicializando e não abre mais nada.

**Comment by elidianaandrade** [comment, marked as answer]
Olá, @Marian97a! Bom dia! Tudo bem? 😊 Eu fiz o login na minha conta e entrou normalmente, caso ainda não consiga peço que entre em contato com o suporte da plataforma através da [Central de Ajuda](https://digitalinnovationone.atlassian.net/servicedesk/customer/portals).

>[!WARNING]
> Gostaria de ressaltar de que este espaço destina-se especificamente a questões técnicas sobre Git, GitHub e iniciativas Open Source. Quaisquer dúvidas relacionadas a plataforma [DIO](https://www.dio.me/) devem ser direcionadas a [Central de Ajuda](https://digitalinnovationone.atlassian.net/servicedesk/customer/portals).
